src/main/java/edu/ksu/canvas/attendance/submitter/AssignmentValidator.java

```diff
@@ -66,11 +66,11 @@ public AttendanceAssignment validateCanvasAssignment(AttendanceAssignment assign
      * Returns true if configuration in the form and in the db is the same
      */
     private boolean isAssignmentConfigurationSaved(AttendanceAssignment assignmentConfigurationFromSetup, AttendanceAssignment attendanceAssignmentSaved) {
-        return assignmentConfigurationFromSetup.getAssignmentPoints().doubleValue() == attendanceAssignmentSaved.getAssignmentPoints().doubleValue()
-                && assignmentConfigurationFromSetup.getExcusedPoints().doubleValue() == attendanceAssignmentSaved.getExcusedPoints().doubleValue()
-                && assignmentConfigurationFromSetup.getAbsentPoints().doubleValue() == attendanceAssignmentSaved.getAbsentPoints().doubleValue()
-                && assignmentConfigurationFromSetup.getTardyPoints().doubleValue() == attendanceAssignmentSaved.getTardyPoints().doubleValue()
-                && assignmentConfigurationFromSetup.getPresentPoints().doubleValue() == attendanceAssignmentSaved.getPresentPoints().doubleValue();
+        return Double.parseDouble(assignmentConfigurationFromSetup.getAssignmentPoints()) == Double.parseDouble(attendanceAssignmentSaved.getAssignmentPoints())
+                && Double.parseDouble(assignmentConfigurationFromSetup.getExcusedPoints()) == Double.parseDouble(attendanceAssignmentSaved.getExcusedPoints())
+                && Double.parseDouble(assignmentConfigurationFromSetup.getAbsentPoints()) == Double.parseDouble(attendanceAssignmentSaved.getAbsentPoints())
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Equality tests should not be made with floating point values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1244)